server/game/core/Constants.ts

```diff
@@ -318,8 +318,9 @@ export enum AbilityRestriction {
 }
 
 export enum StateWatcherName {
+    AttacksThisPhase = 'attacksThisPhase',
```

Comment:
Does this record the attack as a whole, units that were attacked, or units that did the attacking?

Reply:
![image](https://github.com/user-attachments/assets/63c065bf-6616-485d-b058-eb77a8bc3b57)